examples/index.html

```diff
@@ -172,6 +172,7 @@
 
     <ul class="links">
       <li><a href="boilerplate/hello-world/">Hello World</a></li>
+        <li><a href="boilerplate/hello-world-touch-disabled/">Hello World Touch Disabled</a></li>
```

Comment:
It's not necessary to add an example for this, but could you instead add an entry in the look-controls documentation for the new option?

Reply:
Yes, It seems that just a flag does not justify a whole example.

Reply:
Addressed in commit 1d6930b61ac66d080bfe57eada841a032fd6e6e8